app/models/course/material/folder/download.rb

```diff
@@ -0,0 +1,7 @@
+class Course::Material::Folder::Download < ActiveRecord::Base
```

Comment:
Actually do we want to do authorisation for the downloading job ?  

The model is for linking folder and job, thus I can authorise whether user can access the folder before he can download the zip file.  

If authorisation is not needed, I think I can directly use the TrackableJob